### ISSUE TITLE
Improve clarity provided to the user with `hs init` prompts

### ISSUE THREAD

**Issue by miketalley** [HubSpot/hubspot-cms-tools#51]
Based on [this conversation about improving the prompts](https://hubspot.slack.com/archives/CQYN4AM3P/p1578940233005100).

The most confusing prompts seem to be `portalId` and `name`. We need to improve the wording and/or links provided at these prompts to improve the success rate of users when running the `hs init` command.

cc @williamspiro @TheWebTech @gcorne

**Comment by miketalley** [comment]
Thoughts?

### `portalId` prompt
Current
> Enter the HubSpot CMS portal ID:

Proposed
> Enter the Hub ID for your account(this can be found under the DOMAIN column at https://app.hubspot.com/myaccounts-beta)

### `name` prompt
Current
> Enter a name for your portal:

Proposed
> Enter a unique nickname to reference this account(no spaces):

**Comment by williamspiro** [comment]
Both of these are better than what we have now I think, thanks!

I do wonder if we should instruct folks to click on their avatar rather than going to the accounts screen, which feels a little out of context. 

cc @markhazlewood @iamw11y could you both chime on with your recommendations?

**Comment by iamw11y** [comment]
Minor tweaks (BethBot approved)...

> Enter the Hub ID for your account (found under the DOMAIN column at https://app.hubspot.com/myaccounts-beta):

And...

> Enter a unique nickname that will be used to reference this account:

**Comment by erincouse** [comment]
@williamspiro I think "Enter the Hub ID for your account..." is fine. For the second one, I would tweak to say "Enter a unique nickname for this account." I think "unique nickname" makes it pretty clear and makes the second part of the sentence unnecessary.

**Comment by iamw11y** [comment]
A few thoughts as we try to close this out:

1. The use of "unique nickname" means we'll be in conflict with the config file, which refers to this as "name." I'd prefer to keep these terms in sync and think "nickname" is more appropriate, so my preferred course of action would be to update the config file...but something tells me this is easier said than done. That change would be more informative overall, especially for someone who's reading the config file, but an alternative would be to use both in this prompt. Something like, "Enter a unique name that will be used as a nickname to reference this account." Wordy, but sidesteps the terminology conflict.
2. We've debated account ID vs. portal ID, but perhaps Hub ID is an easier path forward, given the general guidance that "portal ID" as for internal use. This would also allow us to remain in sync with other documentation, such as the [Knowledge Base article](https://knowledge.hubspot.com/account/manage-multiple-hubspot-accounts#identify-the-current-account-s-hub-id) linked above. However, we'll again be in a situation where the prompt does not match the config file, which refers to this as a "portal ID."

So in summary, my preferred course of action would be to make the following changes:

- `Enter the Hub ID for your account (the number under the DOMAIN column at https://app.hubspot.com/myaccounts-beta):`
- `Enter a unique nickname that will be used to reference this account:`
- Change the config file to reference `nickname` and `hubID`

But in lieu of making config file changes, I would accept the following (while asking us to consider the above changes going forward):

- `Enter the portal ID for your account (the number under the DOMAIN column at https://app.hubspot.com/myaccounts-beta):`
- `Enter a unique name that will be used as a nickname to reference this account:`

(FWIW, all recommended language above is BethBot approved.)

**Comment by erincouse** [comment]
I think you can cut that sentence a bit by saying either "Enter a unique nickname for referencing your account:" or "Enter a unique name for referencing your account:" -- just my two cents. "to reference your account" is awkward phrasing.

**Comment by iamw11y** [comment]
How about `Enter a unique name to reference your account:` to address that feedback. (Thanks, BTW!) We can use "nickname" in the Getting Started docs, where we have the opportunity to flesh out the explanation for this prompt a bit more.
